family-dashboard/ui/src/main/resources/PhenoTips/FamilySheetMacros.xml

```diff
@@ -49,16 +82,10 @@
 {{html wiki=true clean=false}}
   &lt;table class="family-group-info"&gt;
     &lt;tbody&gt;
-      &lt;tr&gt;
-        &lt;th scope="col" class="button-col"&gt;&lt;/th&gt;
-        &lt;th scope="col"&gt;Group ID&lt;/th&gt;
-        &lt;th scope="col"&gt;Name&lt;/th&gt;
-        &lt;th scope="col"&gt;Description&lt;/th&gt;
-        &lt;th scope="col" class="button-col"&gt;&lt;/th&gt;
-      &lt;/tr&gt;
+      #displayFamilyGroupTableHeader()
       &lt;tr&gt;
         &lt;td&gt;#if ($xcontext.action == 'edit') #displayRemoveButton() #end&lt;/td&gt;
-        &lt;td&gt;&lt;a href="/FamilyGroups/${id}" target="__blank"&gt;$id&lt;/a&gt;&lt;/td&gt;
+        &lt;td&gt;&lt;a href="/FamilyGroups/${id}"&gt;$id&lt;/a&gt;&lt;/td&gt;
```

Comment:
This doesn't work if the application is deployed under a different name. Use `$xwiki.getURL` to get a proper URL.